surveys/2018-09-jupyterlab-ux/README.md

```diff
@@ -0,0 +1,34 @@
+# JupyterLab UX Survey
+
+This folder contains user-testing data collected at JupyterCon 2018 (August 2018) by Cal Poly Interns and Tim George (@tgeorgeux).
+
+The goal of the testing was to answer specific questions about JupyterLab UX. See the [analysis notebook](analysis/questions-to-answer.ipynb) for details.
+
+**How to run the notebook**
+
+
```

Comment:
I was planning on sending a PR this evening in response to a request in the Jupyter Google Group to make this entire repo runnable as a Binder. I think the libs used in the other surveys and this one likely overlap and are the usual culprits.

Reply:
A Binder for this repo would be awesome! It looks like PR #5 includes all the requirements necessary for this survey. Should I remove the requirements/install instructions from this directory?

Reply:
I see no harm in having both. I think of the binder one as a convenience but the requirements listed in each survey subdir as the source of truth.